analysis/tc_tackle_testapp_private.go

```diff
@@ -0,0 +1,104 @@
+	"github.com/konveyor/go-konveyor-tests/hack/addon"
+	"github.com/konveyor/tackle2-hub/api"
+)
+
+var TackleTestappPrivateBinary = TC{
+	Name:        "tackle-testapp-binary",
+	Application: data.TackleTestapp,
+	Identities: []api.Identity{
+		identity.TackleTestappMaven,
+	},
+	Task: Analyze,
+	Labels: addon.Labels{
+		Included: []string{
+			"konveyor.io/target=cloud-readiness",
+		},
+	},
+	Binary: true,
+	Analysis: api.Analysis{
+		Effort: 16,
+		Issues: []api.Issue{
+			{
+				Category:    "mandatory",
+				Description: "Java native libraries (JNI, JNA)",
+				Effort:      7,
+				RuleSet:     "cloud-readiness",
+				Rule:        "jni-native-code-00000",
+				Incidents: []api.Incident{
+					{
+						File:    "/addon/bin/maven/java-project/src/main/java/io/konveyor/demo/config/ApplicationConfiguration.java",
+						Line:    17,
+						Message: "\n Java native libraries might not run in a cloud or container environment.",
+					},
+				},
+			},
+			{
+				Category:    "mandatory",
+				Description: "File system - Java IO",
+				Effort:      1,
+				RuleSet:     "cloud-readiness",
+				Rule:        "local-storage-00001",
+				Incidents: []api.Incident{
+					{
+						File:    "/addon/bin/maven/java-project/src/main/java/io/konveyor/demo/config/ApplicationConfiguration.java",
+						Line:    8,
+						Message: "\n An application running inside a container could lose access to a file in local storage.",
+					},
+					{
+						File:    "/addon/bin/maven/java-project/src/main/java/io/konveyor/demo/config/ApplicationConfiguration.java",
+						Line:    37,
+						Message: "\n An application running inside a container could lose access to a file in local storage.",
+					},
+					{
+						File:    "/addon/bin/maven/java-project/src/main/java/io/konveyor/demo/ordermanagement/config/PersistenceConfig.java",
+						Line:    39,
+						Message: "\n An application running inside a container could lose access to a file in local storage.",
+					},
+					{
+						File:    "/addon/bin/maven/java-project/src/main/java/io/konveyor/demo/ordermanagement/config/PersistenceConfig.java",
+						Line:    40,
+						Message: "\n An application running inside a container could lose access to a file in local storage.",
+					},
+					{
+						File:    "/addon/bin/maven/java-project/src/main/java/io/konveyor/demo/ordermanagement/config/PersistenceConfig.java",
+						Line:    41,
+						Message: "\n An application running inside a container could lose access to a file in local storage.",
+					},
+					{
+						File:    "/addon/bin/maven/java-project/src/main/java/io/konveyor/demo/ordermanagement/config/PersistenceConfig.java",
+						Line:    42,
+						Message: "\n An application running inside a container could lose access to a file in local storage.",
+					},
+					{
+						File:    "/addon/bin/maven/java-project/src/main/java/io/konveyor/demo/ordermanagement/config/PersistenceConfig.java",
+						Line:    61,
```

Comment:
I am afraid these line numbers might keep changing is they modify any rule in future .

Reply:
Correct, but I think that's the purpose of the test - compare analysis results.